.github/workflows/verify.yaml

```diff
@@ -32,9 +32,14 @@ jobs:
         working-directory: go/src/github.com/shipwright-io/build
         args: --timeout=10m
     - name: Install Counterfeiter
+      env:
+        GOPATH: ${GITHUB_WORKSPACE}/go
       run: |
-        go install github.com/maxbrunsfeld/counterfeiter/v6@latest
+        go version
```

Comment:
```suggestion
```

Reply:
Ah, sorry, I missed https://github.com/shipwright-io/build/pull/1091#issuecomment-1214809564 when I started to look into it.

Reply:
You don't need to review the changes in my DEBUG commit. That will go away.